src/win/util.c

```diff
@@ -1380,6 +1380,12 @@ int uv__getpwuid_r(uv_passwd_t* pwd) {
   }
 
   pwd->shell = NULL;
+  pwd->gecos = uv_malloc(1);
+  if (pwd->gecos == NULL)
+    return UV_ENOMEM;
+
+  *(pwd->gecos) = '\0';
```

Comment:
I've kind of gone back on this implementation for windows. I think it makes much more sense to just set it to `NULL` on platforms where `gecos` has no meaning, as is done for `shell`. In the unix case it also doesn't require a separate allocation but I think it's a little wasteful to be doing this for a field that nobody can use, and will probably be ignored.